graphs/possible_bipartition.py

```diff
@@ -1,12 +1,32 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: the worst, O(n^2)
+        Space Complexity: O(n)
     """
-    pass
-
+    if len(dislikes) == 0:
+        return True
+    
+    group1 = set()
+    group1.add(0)
+    group2 = set()
+   
+    q = deque()
+    q.append(1) if len(dislikes[0]) == 0 else q.append(0)
+    visited = {0}
```

Comment:
Similar to line 15.

On line 19, if you end up skipping over the 0th node since it has no neighbors and iterating over the 1st node, you may want to consider adding node 1 to visited before starting your loop. Otherwise, you may end up visiting node 1 twice in the case that it's the neighbor of another node.